src/components/Thanks/SingleVersion/JourneyGeneralPrompt.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+	<div
+		class="tw-rounded md:tw-rounded-lg tw-mx-auto tw-bg-white tw-shadow-lg tw-px-3 md:tw-px-8 tw-py-2 tw-w-full
+            print:tw-shadow-transparent"
+	>
+		<div
+			v-if="loansToDisplay.length && isOptedIn"
+			class="tw-flex tw-items-center tw-justify-center"
+		>
+			<KvUserAvatar
+				v-for="loan, index in loansToDisplay"
+				:key="loan.id"
+				:lender-name="loan?.name"
+				:lender-image-url="loan?.image?.url"
+				class="tw-rounded-full tw-shadow tw-border-white tw-border-2 tw-w-auto"
+				:class="{ 'smaller-borrower-avatar': loansToDisplay.length > 2 && index !== 1 }"
+				:style="{
+					zIndex: index === 1 ? 2 : 1,
+					marginRight: loansToDisplay.length > 2 && index === 0 ? '-22px' : '0',
+					marginLeft: loansToDisplay.length > 1&& index === loansToDisplay.length - 1
+						? '-22px' : '0',
+				}"
+			/>
+		</div>
+		<h2
```

Comment:
I think we want a smaller line-height on this element to match the mocks.

Reply:
Nice call, should we also add this to opt-in module?
<img width="420" alt="Screenshot 2025-02-05 at 3 51 43 p m" src="https://github.com/user-attachments/assets/c63ca500-cfc2-4d16-8b13-fbe71edc7e57" />

Reply:
Ah yup, I was wondering.

Reply:
Is only place we are using opt-in module? Or consider this as a generic change?

Reply:
How about we postpone the opt-in module change. I'll create a ticket, I've been thinking we should create a new component and copy over the code.

Reply:
That makes sense, thanks!